packages/graph/hash_graph/.config/config.toml

```diff
@@ -0,0 +1,102 @@
+[target.'cfg(all())']
```

Comment:
When we say `global` here do we mean global for the hash graph project? Do we intend on sharing these lints with engine?

Reply:
Sorry, yes, for now I mean global for the Graph API workspace. I also intend to share it across the whole repository at some point.